cmd/ocm-backplane/console/console_test.go

```diff
@@ -113,6 +114,37 @@ var _ = Describe("console command", func() {
 		err = utils.CreateTempKubeConfig(&testKubeCfg)
 		Expect(err).To(BeNil())
 	}
+	Context("Should perform existing pods cleanup before starting the console", func() {
+		It("Should not return an error if no pods are found", func() {
+			setupConfig()
+			createPathPodman()
+			mockOcmInterface.EXPECT().GetClusterInfoByID(clusterID).Return(nil, fmt.Errorf("no pods found")).AnyTimes()
```

Comment:
Why we need `GetClusterInfoByID` to return a "no pods found here"?

Reply:
I think we don't need this line because the fake podman we are referring to should have no pod running.